tests/messages/koji/build_complete.json

```diff
@@ -0,0 +1,78 @@
+{
+    "body":{
+        "headers":{
+            "content-length": "864", 
+            "expires": "0", 
+            "old": "BUILDING", 
+            "JMS_AMQP_MESSAGE_FORMAT": "0", 
+            "timestamp": "0", 
+            "JMS_AMQP_NATIVE": "false", 
+            "destination": "/topic/VirtualTopic.eng.brew.build.complete", 
+            "priority": "4", 
+            "subscription": "/queue/Consumer.client-datanommer.upshift-prod.VirtualTopic.eng.>", 
+            "message-id": "ID:messaging.domain.com-41951-1532471159894-4:793878:0:0:1", 
+            "version": "17.4.0", 
+            "attribute": "state", 
+            "release": "8.el8+1325+72a36e76", 
+            "new": "COMPLETE", 
+            "JMS_AMQP_FirstAcquirer": "false", 
+            "type": "BuildStateChange", 
+            "name": "python-attrs"
+        },
+        "msg":{
+            "info": {
+                "package_name": "python-attrs", 
+                "extra": {
+                  "source": {
+                    "original_url": "git://pkgs.domain.com/rpms/python-attrs?#3be3cb33e6432d8392ac3d9e6edffd990f618432"
+                  }
+                }, 
+                "creation_time": "2018-08-03 17:49:42.735506", 
+                "completion_time": null, 
+                "package_id": 63243, 
+                "id": 736244, 
+                "build_id": 736244, 
+                "state": 0, 
```

Comment:
This is the value you're keying on but unfortunately it is incorrect. The state should be 1 to denote it is complete. You could use the state in the "new" key below for the correct value or hard-code it.